site/content/contribute/mobile/build-your-own/android.md

```diff
@@ -72,25 +72,25 @@ To make it easier to customize your build, we've defined a few environment varia
-| `VERSION\_NUMBER `| Set the version of the app at build time to a specific value, rather than using the one set in the project. |  | No |
-| `INCREMENT\_VERSION\_<br>NUMBER\_MESSAGE` | Set the commit message when changing the app version number. | Bump app version number to | No |
-| `INCREMENT\_BUILD\_NUMBER` | Defines if the app build number should be incremented. <br><br>Valid values are: `true`, `false` | `false` | No |
-| `BUILD\_NUMBER` | Set the build number of the app at build time to a specific value, rather than incrementing the last build number. |  | No |
-| `INCREMENT\_BUILD\_<br>NUMBER\_MESSAGE` | Set the commit message when changing the app build number. | Bump app build number to | No |
-| `APP\_NAME` | The name of the app as it is going to be shown on the device home screen. | Mattermost Beta | Yes |
-| `APP\_SCHEME` | The URL naming scheme for the app as used in direct deep links to app content from outside the app. | mattermost | No |
-| `REPLACE\_ASSETS` | Override the assets as described in [White Labeling](contribute/mobile/build-your-own/white-label/). <br><br>Valid values are: `true`, `false` | `false` | No |
-| `MAIN\_APP\_IDENTIFIER` | The package identifier for the app. | | Yes |
-| `BUILD\_FOR\_RELEASE` | Defines if the app should be built in release mode. <br><br>Valid values are: `true`, `false` <br><br>**Make sure you set this value to true if you plan to submit this app Google Play or distribute it in any other way**. | `false` | Yes |
-| `SEPARATE\_APKS` | Build one APK per achitecture (armeabi-v7a, x86, arm64-v8a and x86_64) as well as a universal APK. The advantage is the size of the APK is reduced by about 4MB. <br><br>People will download the correct APK from the Play Store based on the CPU architecture of their device. | `false` | Yes |
-| `SUBMIT\_ANDROID\_TO\_<br>GOOGLE\_PLAY` | Should the app be submitted to the Play Store once it finishes building, use along with `SUPPLY\_TRACK`.<br><br>Valid values are: `true`, `false` | `false` | Yes |
-| `SUPPLY\_TRACK` | The track of the application to use when submitting the app to Google Play Store. Valid values are: `alpha`, `beta`, `production` <br><br>**RIt is not recommended to submit the app to production. First try any of the other tracks and then promote your app using the Google Play console**. | `alpha` | Yes |
-| `SUPPLY\_PACKAGE\_NAME` | The package Id of your application, make sure it matches `MAIN\_APP\_IDENTIFIER`. |  | Yes |
-| `SUPPLY\_JSON\_KEY` | The path to the service account `json` file used to authenticate with Google.<br><br>See the [Supply documentation]( https://docs.fastlane.tools/actions/supply/#setup) to learn more. |  | Yes |
+| `COMMIT_CHANGES_TO_GIT` | Should the Fastlane script ensure that there are no changes to Git before building the app and that every change made during the build is committed back to Git. <br><br>Valid values are: `true`, `false` | `false` | No |
```

Comment:
The COMMIT_CHANGES_TO_GIT description reads awkwardly and isn't clear -- starting the sentence with "Should the..." suggests a question is coming instead. Consider updating for clarity.

Reply:
@cwarnermm I'll leave that to someone who can provide clarification, this PR is only about removing the extra `\` on the variable names.

Reply:
BW #850 is also about the same thing

Reply:
Fair. Approved intended changes via https://github.com/mattermost/mattermost-developer-documentation/pull/850 and noted that a couple of definitions & consistency issues remain.

Reply:
I've reviewed [#850](https://github.com/mattermost/mattermost-developer-documentation/pull/850) and provided some comments. Perhaps we can sync both PRs up to include my feedback and Carrie's - that way they'll be aligned in terms of tense and so on. Happy to do that if it works for everyone.

Reply:
That would be great, @justinegeffen -- Thank you so much! May I transfer https://mattermost.atlassian.net/browse/MM-36397 to you for completion?